### ISSUE TITLE
Impedance gains sliders

### ISSUE THREAD

**Issue by alaurenzi** [ADVRHumanoids/XBotGUI#12]
Hi Alessandro,
it would be nice in the future to also have gains for impedance gains in addittion to joint position!
What do you think?

**Comment by alaurenzi** [comment]
For some reason I cannot put labels o.O would be an enhancement!

**Comment by alessandrosettimi** [comment]
Indeed! There is already issue #6 that is about torque control, then @liesrock also pointed out that we should add stiffness and damping control. So, what is the most useful thing to do first? Another thing, are some of this gains adjustable with position control? It would be nice to have a list of possible controls and which gains you can adjust with a certain control.

**Comment by alessandrosettimi** [comment]
Seems that ADVR developers had only read permissions on this repo... fixed!

**Comment by liesrock** [comment]
No! They have write and you (@alessandrosettimi) are the admin!

**Comment by liesrock** [comment]
Probably the priority is the integration with the walking module: during the sprint week we can speak in person about the list of possible control by the GUI!

**Comment by alessandrosettimi** [comment]
@liesrock yes! Because I fixed It xD

However, right now I'm working on walking module related tasks. It is ok to me to speak in person during the sprint week for the control side.

**Comment by liesrock** [comment]
:D sorry! That's why you are the Admin 👍

**Comment by alaurenzi** [comment]
@pepperig  This is a super-useful feature that you can also implement on your web gui!

**Comment by pepperig** [comment]
Yes! it was already in my mind :-) However in that case can work just from NRT
